code/modules/atmospherics/machinery/trinary/manifold_valve.dm

```diff
@@ -0,0 +1,140 @@
+
+/obj/machinery/atmospherics/trinary/manifold_valve
+	icon = 'icons/obj/atmospherics/manifold_valve.dmi'
+	icon_state = "manifold_valve0"
+
+	name = "manifold valve"
+	desc = "A pipe valve"
+
+	var/divert = FALSE
```

Comment:
might be worth doc commenting what divert does

Reply:
ight